apps/passport-server/test/frogcryptodb.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { Biome } from "@pcd/eddsa-frog-pcd";
+import chai, { expect } from "chai";
+import chaiAsPromised from "chai-as-promised";
+import "mocha";
+import { step } from "mocha-steps";
+import { Client } from "pg";
+import { Pool } from "postgres-pool";
+import { getDB } from "../src/database/postgresPool";
+import {
+  fetchUserFeedsState,
+  getFrogData,
+  initializeUserFeedState,
+  insertFrogData,
+  sampleFrogData,
+  updateUserFeedState
+} from "../src/database/queries/frogcrypto";
+import { overrideEnvironment, testingEnv } from "./util/env";
+import { testFrogs } from "./util/frogcrypto";
+
+describe("database reads and writes for frogcrypto features", function () {
+  this.timeout(15_000);
+
+  let db: Pool;
+  let client: Client;
+
+  this.beforeAll(async () => {
+    chai.use(chaiAsPromised);
```

Comment:
i don't think this needs to be done, because as far as I understand the `globalhooks.spec.ts` file takes care of initializing this lib

Reply:
After commenting this out, I get `     Error: Invalid Chai property: rejectedWith`. I can look into this separately.